.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true
```

Comment:
Are you using mocha for testing? Why?

Reply:
I think there is a separate repo for the React front-end so this linting is basically for the backend, and I used the rules Frank gave in the PT board.  About using mocha, we thought since its widely used, it will be easier to look for resources when we encounter problems.